setup.cfg

```diff
@@ -18,4 +18,4 @@ exclude=.tox,venv,awx/lib/site-packages,awx/plugins/inventory/ec2.py,awx/plugins
 [flake8]
 max-line-length=160
 ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,W291,W391,W293,E731,W504
-exclude=.tox,venv,awx/lib/site-packages,awx/plugins/inventory,awx/ui,awx/api/urls.py,awx/main/migrations,awx/main/tests/data,node_modules/,awx/projects/,tools/docker,awx/settings/local_*.py,installer/openshift/settings.py,build/,installer/,awxkit/test
+exclude=.tox,venv,awx/lib/site-packages,awx/plugins/inventory,awx/ui,awx/api/urls.py,awx/main/migrations,awx/main/tests/data,node_modules/,awx/projects/,tools/docker,awx/settings/local_*.py,installer/openshift/settings.py,build/,installer/,awxkit/test,awx_modules/
```

Comment:
I have a `setup.cfg` file for the modules, and it is passing flake8.

The problem here is that I can't use different `setup.cfg` files for different folders. I still need to figure this out.

Ideally, I would just add the flake8 command for the modules to the `make flake8` command. But this command has an extremely non-obvious configuration and I frankly don't think this can be done cleanly.

That leaves with the option of maybe adding it to `make prepare_test_modules` or `make test_modules`. The overall configuration of these targets still needs a finalized design.

Reply:
Got it, I was just making sure we were not caring about those files.